programs/lz4cli.c

```diff
@@ -134,7 +134,8 @@ static int usage_advanced(const char* exeName)
     DISPLAY( " -r     : operate recursively on directories (sets also -m) \n");
 #endif
     DISPLAY( " -l     : compress using Legacy format (Linux kernel compression)\n");
-    DISPLAY( " -B#    : Block size [4-7] (default : 7) \n");
+    DISPLAY( " -B#    : cut file into independent blocks of size # bytes [32+] \n");
```

Comment:
It's not necessity for these blocks to be "independent".
Block dependency is controlled by another command, `-BD`.
It should be possible to mix it with custom block sizes.

Reply:
I removed the "independent"; I also did a manual test with "lz4 -BD" to make sure nothing breaks.